src/plugin/nodeSpec.ts

```diff
@@ -365,21 +374,82 @@ export const createNodesForFieldValues = <
         return node;
       }
       case FieldContentType.NESTED: {
-        let content = fieldValue as string;
-        return [
-          field.type === "nestedElement"
-            ? createContentNodeFromRichText(
-                schema,
-                content,
-                nodeType.create({ type: field.type })
-              )
-            : createContentNodeFromText(content, field, nodeType),
-        ];
+        const content = fieldValue as unknown[];
+        const node = createNestedElementNode(
+          content,
+          field,
+          nodeType,
+          schema,
+          getNodeFromElementData,
+          transformElementIn
+        );
+
+        if (!node) {
+          console.log({ content, field, nodeType, nodeName });
```

Comment:
did you mean to leave this in? if so, in that it throws after, perhaps...
```suggestion
          console.error({ content, field, nodeType, nodeName });
```